module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -542,6 +544,11 @@ protected function chooseCallNumber($hCallNumP, $hCallNum, $iCallNumP, $iCallNum
      */
     public function getHolding($bibId, array $patron = null, array $options = [])
     {
+        $showDueDate = $this->config['Availability']['showDueDate'] ?? true;
+        $showTime = $this->config['Availability']['showTime'] ?? true;
```

Comment:
Maybe the default in the code should match the default in Folio.ini for consistency.

Reply:
Absolutely, sorry that this mismatched - I'll set showTime to false by default